CIP-0027.md

```diff
@@ -0,0 +1,74 @@
+## Simple Summary
+
+Create a standard tagging for royalties&#39; distribution within the metadata of Cardano Assets.
+
+## Abstract
+
+This proposed tagging standard will allow for uniform royalties&#39; distributions. It has been developed with input from Artano, CNFT.io, Digital Syndicate, and Buffy Bot.
+
+## Motivation
+
+There is a significant number of artists who have expressed desire to leverage a royalties function within Cardano Assets, which will allow them a percentage of the future resale of assets they have created. There is also a misperception that Smart Contracts will have a built in solution for royalties. As of the writing of this CIPS proposal, Smart Contracts do not have this capability. By creating a standard for metadata tags, it will allow for immediate adoption, and backwards compatibility.
+
+## Specification
+
+&quot;royalties&quot; - Placed within CNFT Metadata.
+ Can either be an array that contains the royalties instructions or contains a link to offchain royalties json which includes the royalties instructions. The link to an offchain json is recommended so that it can be modified in the future.
+
+Royalties instructions tags:
+
+&quot;royaltiesContact&quot;: contact info for royalties inquiries (optional)
+ &quot;royaltiesInstructionVersion&quot;: version of formatting used
+ &quot;royaltiesPercent&quot;: Overall percentage of resale to be distributed as royalties
+ &quot;royaltiesPrimaryOwner&quot;: Friendly name of first royalties party (optional)
+ &quot;royaltiesPrimaryPercent&quot;: Percentage of royalties distribution for the first party
+ &quot;royaltiesPrimaryWallet&quot;: Wallet address to forward royalties to first party
+ &quot;royaltiesSecondaryOwner&quot;: Friendly name of second royalties party (optional)
+ &quot;royaltiesSecondaryPercent&quot;: Percentage of royalties distribution for the second party (optional)
+ &quot;royaltiesSecondaryWallet&quot;: Wallet address to forward royalties to second party (optional)
+ &quot;royaltiesTertiaryOwner&quot;: Friendly name of third royalties party (optional)
+ &quot;royaltiesTertiaryPercent&quot;: Percentage of royalties distribution for the third party (optional)
+ &quot;royaltiesTertiaryWallet&quot;: Wallet address to forward royalties to third party (optional)
+ &quot;royaltiesQuaternaryOwner&quot;: Friendly name of fourth royalties party (optional)
+ &quot;royaltiesQuaternaryPercent&quot;: Percentage of royalties distribution for the fourth party (optional)
+ &quot;royaltiesQuaternaryWallet&quot;: Wallet address to forward royalties to fourth party (optional)
+ 
+## Example JSON
+
+Onchain
+
+"royalties": "https://bit.ly/3yToIri"
+   
+Offchain
+
+[{
```

Comment:
Please use: "```" for formatted text and I also formatted this for you.

```
[
    {
        "royaltiesActive": "true",
        "royaltiesContact": "loa-royalties@digitalsyndicate.io",
        "royaltiesInstructionVersion": "0.1",
        "royaltiesPercent": "10.0",
        "royaltiesPrimaryPercent": "40.0",
        "royaltiesPrimaryWallet": "addr1v80gf3xn485sc9qea4keh99klqk70vhdd923c43kh3tud9g2gmu0w",
        "royaltiesPrimaryOwner": "Praga Khan",
        "royaltiesSecondaryPercent": "40.0",
        "royaltiesSecondaryOwner": "Karl Kotas",
        "royaltiesSecondaryWallet": "addr1v98us4rxzjwxm0w0sl22u64tdw0c0vclufj2qwacgf5dcaqpn85q8",
        "royaltiesTertiaryPercent": "20.0",
        "royaltiesTertiaryOwner": "DigitalSyndicate",
        "royaltiesTertiaryWallet": "addr1vxjrmfpz9eh8ypps8yfh50snuk2e4q0ek60lxgcrt9y3tucxs2yfw"
    }
]
```

Reply:
Thank you very much!